docs/artifact-plugin.md

```diff
@@ -11,30 +11,19 @@ By implementing a plugin, you can integrate with proprietary storage systems, ad
 
 To create an artifact plugin, you need to:
 
-### 1. Create and Distribute a Docker Image
-
-Your plugin must be packaged as a Docker image that contains:
-
-- Your plugin implementation
-- All necessary dependencies and runtime requirements
-- The GRPC server that implements the artifact interface
-
-The Docker image will be deployed alongside workflow pods as sidecars and init containers.
-
-### 2. Implement a GRPC Server
+### 1. Implement a GRPC Server
 
 Your plugin's entrypoint must run a GRPC server that:
 
+<!-- TODO: specify path format, including `unix:` or not? -->
```

Comment:
@Joibel this was a question for you because the "spec" seemed a little vague:

> Listens on the socket path provided as the first and only command-line parameter

This doesn't specify the format of the parameter, and having never used sockets before I assumed from clues in the `Start Your Plugin Server` section that I would pass it in as a plain file path. Was it intentional (and correct) to _not_ add the `unix:` prefix?

Reply:
Discussed live - it will be handled by the server program. Some will add automatically (Go tools). Adding manually is a case of `if not server.startswith("unix:"): server = "unix:" + server` in Python.